docs/guides/SimpleStartupGuide.md

```diff
@@ -0,0 +1,396 @@
+   ```bash
+   newgrp docker 
+   ```
+
+4. To verify the Docker installation, run `hello-world`:
+
+   ```bash
+   docker run hello-world
+   ```
+   A **Hello from Docker!** greeting indicates successful installation.
+
+   ```bash
+   Unable to find image 'hello-world:latest' locally
+   latest: Pulling from library/hello-world
+   2db29710123e: Pull complete 
+   Digest: sha256:10d7d58d5ebd2a652f4d93fdd86da8f265f5318c6a73cc5b6a9798ff6d2b2e67
+   Status: Downloaded newer image for hello-world:latest
+
+   Hello from Docker!
+   This message shows that your installation appears to be working correctly.
+   ...
+   ```
+
+### Install Docker Compose
+Install Docker from the official repository as documented on the [Docker Compose](https://docs.docker.com/compose/install/#install-compose) site. See the Linux tab. 
+
+1. Download current stable Docker Compose:
+   ```bash
+   sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+   ```
+   >**NOTE:** When this guide was created, version 1.29.2 was current.
+
+2. Set permissions:
+   ```bash
+   sudo chmod +x /usr/local/bin/docker-compose
+   ```
+
+###  Download EdgeX Compose
+Clone the EdgeX compose repository
+
+   ```bash
+   git clone https://github.com/edgexfoundry/edgex-compose.git
+   ```
+
+### Install Tools
+Install the media utility tool:
+
+   ```bash
+   sudo apt install build-essential jq curl mplayer v4l-utils
+   ```
+
+### Tool Descriptions
+The table below lists command line tools this guide uses to help with EdgeX configuration and device setup.
+
+| Tool        | Description | Note |
+| ----------- | ----------- |----------- |
+| **build-essential** |  Developer tools such as libc, gcc, g++ and make. | |
+| **jq**   |Parses the JSON object returned from the `curl` requests. |The `jq` command includes parameters that are used to parse and format data. In this tutorial, the `jq` command has been configured to return and format appropriate data for each `curl` command that is piped into it. |
+| **curl**     | Allows the user to connect to services such as EdgeX. |Use curl to get transfer information either to or from this service. In the tutorial, use `curl` to communicate with the EdgeX API. The call will return a JSON object.|
+| **mplayer** |  used to view the video stream | |
+| **v4l-utils** | used to determine the video stream path of a usb camera | |
+
+>Table 1: Command Line Tools
+## Get the Source Code
+###  Download EdgeX Compose Repository
+
+1. Create a directory for the EdgeX compose repository:
+   ```bash
+   mkdir ~/edgex
+   ```
+
+2. Change into newly created directory:
+   ```bash
+   cd ~/edgex
+   ```
+
+3. Clone the EdgeX compose repository
+   ```bash
+   git clone https://github.com/edgexfoundry/edgex-compose.git
+   ```
+
+
+## Run the Service
+
+1. Navigate to the Edgex compose directory.
+
+   ```shell
+   cd ~/edgex/edgex-compose/compose-builder
```

Comment:
I don't think the user needs to go into the final `compose-builder` path if using the default images.